packages/strapi-admin/admin/src/components/LeftMenuLinkSection/index.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import matchSorter from 'match-sorter';
+import styled from 'styled-components';
+import { sortBy } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+
+import LeftMenuLink from '../LeftMenuLink';
+import LeftMenuLinkHeader from '../LeftMenuLinkHeader';
+
+const LeftMenuListLink = styled.div`
+  max-height: 180px;
+  overflow: auto;
+`;
+const EmptyLinksList = styled.div`
```

Comment:
Same here about putting the components in a separate file. Maybe you can create a sub folder for them?